Makefile

```diff
@@ -45,6 +45,10 @@ check: | build verify
 # Example:
 #   make verify
 verify: build
+	# Verify extended tests compile
+	$(MAKE) build WHAT=test/extended/extended.test
```

Comment:
hack/build-go.sh test/extended/extended.test test/extended/networking/extended.test is preferred here